src/fftvis/core/antenna_gridding.py

```diff
@@ -142,62 +118,43 @@ def check_antpos_griddability(
     max_denominator: int = 10**6,
     max_factor: int = 1000,
 ) -> Tuple[bool, Dict[Any, np.ndarray], np.ndarray]:
-    """
-    Check if the antenna positions can be gridded. This function checks if the
-    antenna positions can be scaled to integers using a linear transformation.
-    It first infers the 2D lattice basis from the antenna positions, then checks
-    if the positions can be scaled to integers. If they can, it returns the
-    modified antenna positions and the transformation matrix. If not, it returns
-    the original antenna positions and an identity transformation matrix.
+    """Check if antenna positions can be gridded.
 
     Parameters
     ----------
     antpos : dict
-        Dictionary of antenna positions in the form {ant_index: np.array([x,y,z])}
-        where the antenna positinons are in units of meters.
+        Antenna positions {ant_index: np.array([x,y,z])} in meters.
     tol : float
-        Tolerance for checking if values are close to integers in units of meters.
+        Tolerance in meters.
     max_denominator : int
-        Maximum denominator for the fractions.
+        Maximum denominator.
     max_factor : int
-        Maximum allowed factor for scaling. If the factor exceeds this value,
-        the function will return False.
+        Maximum scaling factor.
 
-    Returns:
+    Returns
     -------
-    is_griddable : bool
-        True if the antenna positions can be gridded, False otherwise.
-    modified_antpos : dict
-        The modified antenna positions after scaling.
-    transform_matrix : np.ndarray
-        The transformation matrix used to scale the antenna positions.
-        This is a 3x3 matrix that transforms the original coordinates
-        to the grid coordinates.
+    tuple
+        (is_griddable, modified_antpos, transform_matrix)
```

Comment:
This is not the correct format for documenting return values... the original is correct.